src/modules/claims/claims.service.ts

```diff
@@ -1320,4 +1346,24 @@ export class ClaimsService {
       await this._signerService.signMessage(arrayify(proofHash))
     );
   }
+
+  private _setClaimIssuerVerifier() {
+    const credentialResolver: CredentialResolver = new IpfsCredentialResolver(
+      this._signerService.provider,
+      this._didRegistry.registrySettings,
+      this._didRegistry.ipfsStore
+    );
+    const domainReader = new DomainReader({ensRegistryAddress: chainConfigs()[this._signerService.chainId].ensResolverV2Address, provider: this._signerService.provider})
```

Comment:
```suggestion
    const domainReader = new DomainReader({ensRegistryAddress: chainConfigs()[this._signerService.chainId].ensRegistryAddress, provider: this._signerService.provider})
```

Reply:
@whitneypurdum @JGiter Am I right with the above suggested change? Also, we need to `addKnownResolver` for the domainReader to be able to resolve namespaces.

Reply:
Yes, both your comments are correct. Actually I think that instead of creating domain reader here it should be exported from domains service or created by issuer verifier

Reply:
Yeah, exporting from `DomainService` sounds good to me since we already have it at `ClaimService` creation https://github.com/energywebfoundation/iam-client-lib/blob/4f3688e5329ec1ccb222a90049583c6633922fb3/src/modules/claims/claims.service.ts#L87

Reply:
@whitneypurdum If you could make the above suggested changes, it would be great.

Reply:
Sure thing @nichonien